libs/ui/src/reports/admin_tally_report.tsx

```diff
@@ -60,54 +59,52 @@ export function AdminTallyReport({
   };
 
   return (
-    <ThemeProvider theme={tallyReportThemeFn}>
-      <TallyReport data-testid={testId}>
-        <ReportSection>
```

Comment:
I know that combining these doesn't change the styling due to VxAdmin PDF snapshots remaining stable.